src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/ClassDataAbstractionCouplingCheck.java

```diff
@@ -107,86 +107,189 @@
  * &lt;module name="ClassDataAbstractionCoupling"/&gt;
  * </pre>
  * <p>
- * To configure the check with a threshold of 5:
+ * Code examples:
+ * </p>
+ * <p>
+ * The check passes without violations in the following.
  * </p>
  * <pre>
- * &lt;module name="ClassDataAbstractionCoupling"&gt;
- *   &lt;property name="max" value="5"/&gt;
- * &lt;/module&gt;
+ * class InputClassCoupling {
+ *   Set set = new HashSet(); // Ignored due to default excludedClasses property
+ *   Map map = new HashMap(); // Ignored due to default excludedClasses property
+ *   Date date = new Date(); // Counted, 1
+ *   Time time = new Time(); // Counted, 2
+ *   Place place = new Place(); // Counted, 3
+ * }
  * </pre>
  * <p>
- * To configure the check with two excluded classes {@code HashMap} and {@code HashSet}:
+ * The check results in a violation in the following.
  * </p>
  * <pre>
- * &lt;module name="ClassDataAbstractionCoupling"&gt;
- *   &lt;property name="excludedClasses" value="HashMap, HashSet"/&gt;
- * &lt;/module&gt;
+ * class InputClassCoupling {
```

Comment:
Please remove and update the above example. There should be a single example for default configuration with a violation for default values.

Reply:
https://github.com/checkstyle/checkstyle/blob/a231f5202c6769f9c664eb53a1bb370ec8788c96/src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/ClassFanOutComplexityCheck.java#L97
The default config had 2 examples in this, one with and one without violations.

Reply:
Good point. Thanks for the link. Let is keep consistent.